### ISSUE TITLE
WHILE walk into Cond

### ISSUE THREAD

**Issue by larshp** [abaplint/abaplint#1195]
```abap
DATA lv_int1 TYPE i.
DATA lv_int2 TYPE i.
WHILE lv_int1 < lv_int2.
ENDWHILE.
```

should report zero unused variables
